README.md

```diff
@@ -44,7 +44,7 @@ Generic File Submission API (to be used by the Family Law Act Application at fir
 ## Running the App
 
 By default a demo mode is enabled.
-if you want to integrate with the CSO application create a local `.env` at the root of the repository based off `.env.template`. 
+if you want to integrate with the CSO application create a local `.env` at the root of the repository based off `.env.template`.
```

Comment:
...with the CSO application, first create...

suggestion, non-blocking

Reply:
done ;-)